### ISSUE TITLE
Some panels empty.

### ISSUE THREAD

**Issue by landjea** [sebklaus/profiler#6]
Using a base install of Laravel 4.0.9, but storing sessions in a mysql db, the routes and sql panels are both empty. Routes says that 2 routes are in use, but the panel is blank.

HTML for the panel....doubt it helps though.

```
<div class="anbu-tab-pane anbu-table anbu-routes">
                <table>
    <tr>
        <th></th>
        <th>Path</th>
        <th>Route</th>
        <th>Uses</th>
        <th>Before</th>
        <th>After</th>
    </tr>
                                </tr>
                            </tr>
    </table>
            </div>
```

 SQL, which I am using for Sessions, comes up with the "no SQL queries were executed" message.

**Comment by landjea** [comment]
ROUTES : I think the problem comes from the _routes.blade.php - you are using "if STRIPOS". But the starting position for both of those statements is "0", which evaluates to FALSE. You just need to rework that bit to get the routes to show up.

 I am not at home to do a pull request for it.

**Comment by landjea** [comment]
And, to test my thoughts, I added a "&lt;tr&gt;&lt;td&gt;Hello&lt;/td&gt;&lt;/tr&gt;" to the foreach, and then each of the if statements. It only appears in the foreach so neither of the if statements fire.

I changed the stripos check to ".1." and ".0." to see if that will fire the code since the new return should be 1, and the following error pops up : 

```
[2013-11-14 01:21:41] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException'
 with message 'Call to undefined method Illuminate\Routing\Router::current()' in 
C:\wamp\www\wrassle\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php:203
```

I then removed the second statement (@elseif) completely to force the code to run and that same error appears.

So there is a second problem.

**Comment by landjea** [comment]
I wrote a separate DB insert into the sessions table and that one appears, but it doesn't pick up the Session DB insert for whatever reason. So I guess that _is_ working but I would have thought that all queries would be logged in the count. Seems like Laravel isn't logging that one for the profiler to be able to pick up.

**Comment by sebklaus** [comment]
Hi landjea,

I am very sorry for the long wait regarding this issue.
I included your Pull Request, as you pointed out the issue. I have adjusted the if statement to check for the full Laravel version and included the missing boolean comparison. I also changed the used API function for version 4.0.x., bringing back the actual content of the panel. Don't know what I was thinking, when I released that!

Hopefully all issues are now resolved.
Cheers, Seb
